src/components/__tests__/Panel.js

```diff
@@ -30,6 +30,21 @@ describe('default render', () => {
     expect(wrapper.html()).toMatchSnapshot();
   });
 
+  it('passes data without refinement', () => {
+    const defaultScopedSlot = jest.fn();
+    const wrapper = mount(Panel, {
+      scopedSlots: {
+        default: defaultScopedSlot,
+      },
+    });
+
+    wrapper.setData({
+      canRefine: false,
+    });
+
+    expect(defaultScopedSlot).toHaveBeenCalledWith({ noRefinement: true });
```

Comment:
Would it make sense to test that it's been called once?

Reply:
I don't think it's relevant, we can call the slot multiple times but it wouldn't change anything for users